### PR TITLE
(DOCSP-45676), (DOCSP-45679), (DOCSP-45680): Updated Local Deployment page.

### PR DESCRIPTION
**Pull request by corryroot**
@fmenezes, I've drafted all the updated for the Local Deployment page. Can you provide more information about the `--initdb` option? Where do the `.js` and `.sh` files come from?

- [DOCSP-45676](https://jira.mongodb.org/browse/45676)
- [DOCSP-45679](https://jira.mongodb.org/browse/45679)
- [DOCSP-45680](https://jira.mongodb.org/browse/45680)
- [STAGING](https://deploy-preview-784--docs-atlas-cli.netlify.app/atlas-cli-deploy-local/#move-a-local-atlas-deployment-to-a-cloud-atlas-deployment)

### Self-Review Checklist

- [ ] Check that the submodule pulled in the right changes (if applicable).
- [ ] [Define](https://wiki.corp.mongodb.com/display/DE/Taxonomy+tagging+instructions) taxonomy [values](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) at top of page.
- [ ] Add genre facets (tutorial or reference), as in this [example PR](https://github.com/10gen/cloud-docs/pull/5042).
- [ ] Add programmingLanguage (if necessary).
- [ ] Add meta keywords (if necessary).
- [ ] Resolve any new warnings or errors in the build.
- [ ] Proofread for spelling and grammatical errors.
- [ ] Check staging for rendering issues.
- [ ] Confirm links are working.

### CODE REVIEW

File: source/atlas-cli-deploy-local.txt [hunk 1]
```diff
@@ -119,6 +119,22 @@ mode.
...
+            a. Copy the following command:
+
+               .. code-block:: sh
+
+                  atlas deployments setup --initdb {folder}
+
+            #. Replace the ``{folder}`` placeholder with the directory 
+               that contains the ``.js`` and ``.sh`` files to run 
+               inside the local container in alphanumeric order with 
+               ``mongosh`` and ``bash`` respectively.
```
Comment (fmenezes) [inline source/atlas-cli-deploy-local.txt:134]:
```suggestion
               inside the local container in alphanumeric order
```

File: source/atlas-cli-deploy-local.txt [hunk 2]
```diff
@@ -119,6 +119,22 @@ mode.
...
+            
+            a. Copy the following command:
+
+               .. code-block:: sh
+
+                  atlas deployments setup --initdb {folder}
+
+            #. Replace the ``{folder}`` placeholder with the directory 
+               that contains the ``.js`` and ``.sh`` files to run 
+               inside the local container in alphanumeric order with
```
Comment (fmenezes) [inline source/atlas-cli-deploy-local.txt:133]: I don't think we need the next line

```suggestion
               inside the local container in alphanumeric order.
```

File: source/atlas-cli-deploy-local.txt [hunk 3]
```diff
@@ -182,6 +198,22 @@ mode.
...
```
Comment (fmenezes) [inline source/atlas-cli-deploy-local.txt:212]: same comment as above

File: source/atlas-cli-deploy-local.txt [hunk 4]
```diff
@@ -296,6 +328,22 @@ mode.
...
+            
+            a. Copy the following command:
+
+               .. code-block:: sh
+
+                  atlas deployments setup myLocalRs1 --type local --force --initdb {folder}
+
+            #. Replace the ``{folder}`` placeholder with the directory 
+               that contains the ``.js`` and ``.sh`` files to run 
+               inside the local container in alphanumeric order with
```
Comment (fmenezes) [inline source/atlas-cli-deploy-local.txt:343]: same

File: source/atlas-cli-deploy-local.txt [hunk 5]
```diff
@@ -403,6 +451,182 @@ atlas deployments respectively.
...
+            
+      a. Copy the following command:
+
+         .. code-block:: sh
+
+            atlas deployments setup --type local --initdb {folder}
+
+      #. Replace the ``{folder}`` placeholder with the directory 
+         that contains the ``.js`` and ``.sh`` files to run 
+         inside the local container in alphanumeric order with
```
Comment (fmenezes) [inline source/atlas-cli-deploy-local.txt:489]: same

File: source/atlas-cli-deploy-local.txt [hunk 6]
```diff
@@ -403,6 +451,182 @@ atlas deployments respectively.
...
```
Comment (fmenezes) [inline source/atlas-cli-deploy-local.txt:574]: same

File: source/atlas-cli-deploy-local.txt [hunk 7]
```diff
@@ -403,6 +448,180 @@ atlas deployments respectively.
...
+         atlas deployments delete --type local
+
+.. _atlas-cli-update-local-deployment:
+
+Update a Local Atlas Deployment
+-------------------------------
+
+You can use `Docker <https://www.docker.com/>`__ and 
+:dbtools:`MongoDB Database Tools </installation/installation/>` to 
+update a local |service| deployment to newer version of the image.
```
Comment (erabil-mdb) [inline source/atlas-cli-deploy-local.txt:548]:
```suggestion
update a local |service| deployment to a newer version of the image.
```

**Review by erabil-mdb — APPROVED** [review]
LGTM % tiny typo

**Review by fmenezes — APPROVED** [review]

**Pull request merged**